apps/prairielearn/assets/scripts/instructorAssessmentManualGradingAssessmentClient.ts

```diff
@@ -0,0 +1,15 @@
+import { onDocumentReady } from '@prairielearn/browser-utils';
+
+onDocumentReady(() => {
+  document.querySelectorAll<HTMLButtonElement>('.js-assign-grader-btn').forEach((button) =>
+    button.addEventListener('click', () => {
+      const assessmentQuestionInput = document.querySelector<HTMLInputElement>(
+        '#grader-assignment-modal input[name="unsafe_assessment_question_id"]',
+      );
+      if (assessmentQuestionInput) {
+        assessmentQuestionInput.value = button.dataset.assessmentQuestionId ?? '';
+      }
+      $('#grader-assignment-modal').modal('show');
+    }),
+  );
+});
```

Comment:
Nit, consider writing this with [`templateFromAttributes`](https://github.com/PrairieLearn/PrairieLearn/blob/master/packages/browser-utils/README.md#templatefromattributes) (we use this everywhere in PT where we need to template a modal, and I'd like to try to do the same in PL):

```suggestion
onDocumentReady(() => {
  const modal = document.getElementById('#grader-assignment-modal');
  document.querySelectorAll<HTMLButtonElement>('.js-assign-grader-btn').forEach((button) =>
    button.addEventListener('click', () => {
      templateFromAttributes(button, modal, {
        'data-assessment-question-id': 'input[name="unsafe_assessment_question_id"]',
      });
      $('#grader-assignment-modal').modal('show');
    }),
  );
});
```

Also consider opening the modal with the data attribute API (`data-toggle`, `data-target`, and such).

Ideally we'd do event handling with `delegated-events`, but we need BS5 to be universally used first.

Reply:
Switched to data-toggle in af59831b8c26ae32d42bbf34fb2b5659fbdf55cc.